MobileBuy/sample/src/main/java/com/shopify/sample/util/Executors.java

```diff
@@ -0,0 +1,21 @@
+package com.shopify.sample.util;
+
+import android.os.Handler;
+import android.os.Looper;
+
+public class Executors {
+
+  private final Handler mainThreadHandler;
+
+  public static Executors getDefault() {
+    return new Executors(new Handler(Looper.getMainLooper()));
+  }
+
+  private Executors(Handler mainThreadHandler) {
```

Comment:
I would call this `CallbackExecutor` instead

Reply:
Sure.